### PR TITLE
gui_qt/paper_tape: fix save feature

### LINKED ISSUE #1433: Plover: Paper Tape window exports no stroke data
**Describe the bug**

The Plover: Paper Tape window only exports a blank file when saving it to a file. It will be 0 bytes with no stroke data despite strokes being logged on the Paper Tape window.

**To Reproduce**

1. Have the _Plover: Paper Tape_ window open.
2. If there are no strokes logged on the Paper Tape window yet, input some strokes before continuing.
3. Click the 'Save' icon located at the bottom of the _Plover: Paper Tape_ window.
4. Choose a folder to save the file and click _Save_.
5. Navigate to the folder the file was saved to.

**Expected behavior**

The output that the Paper Tape window produces should match the data in the `steno-notes-YYYY-MM-DD-HH-MM.txt` file.

**Screenshots**

Not applicable.

**Operating system**

 - OS: Windows 10
 - Plover Version: Plover 4.0.0.dev10+115.gc75215e                

**Hardware**

Not applicable.

### PR DESCRIPTION
**Pull request by benoit-pierre**
## Summary of changes

Fix regression introduced in #1308.

Closes #1433.

### Pull Request Checklist
- [ ] Changes have tests
- [x] News fragment added in news.d. See [documentation](https://github.com/openstenoproject/plover/blob/master/doc/developer_guide.md#making-a-pull-request) for details

### CODE REVIEW

File: news.d/feature/1434.ui.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Improve accessibility:
+- Disable tag-key navigation in tables, so focusing a table does not lock global tab-key navigation to it.
```
Comment (morinted) [inline news.d/feature/1434.ui.md:2]: tag-key typo?

File: news.d/feature/1434.ui.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Remove some container widget, tweak focus rules to avoid extra unnecessary steps when using tab-key navigation (like selecting the dictionaries widget outer frame).
```
Comment (morinted) [inline news.d/feature/1434.ui.md:3]: some container widget**s**

**Review by morinted — CHANGES_REQUESTED** [review]

**Review by morinted — APPROVED** [review]
It's unclear to me, why is the news entry repeated for each feature?

**Comment by benoit-pierre** [comment]
> It's unclear to me, why is the news entry repeated for each feature?

Because the work was split in 3 different PRs. Those will be merged automatically by towncrier:

```
- Improve accessibility:
[…]
- Use lists for the dictionaries widget, suggestions widget, and the paper tape. (#1308, #1332, #1434)
```

 And I've not made this PR a different bugfix entry because the changelog is still describing the differences with the latest non-continuous release (v4.0.0.dev10).

**Comment by morinted** [comment]
Thanks. That helps and of course it doesn't make sense to say we fixed something that was never release.

**Pull request merged**